<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CRBComboBox" name="cmbServiceGroup"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextEdit" name="edtName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>&amp;Наименование</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblServiceGroup">
     <property name="text">
      <string>&amp;Группа</string>
     </property>
     <property name="buddy">
      <cstring>cmbServiceGroup</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>2</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInfo">
      <attribute name="title">
       <string>Основная информация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="10" column="1">
        <widget class="QDoubleSpinBox" name="edtUetAdultDoctor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QDoubleSpinBox" name="edtSuperviseComplexityFactor">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="edtQualityLevel">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>999999.998999999952503</double>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblQualityLevel">
         <property name="text">
          <string>Уровень качества лечения</string>
         </property>
         <property name="buddy">
          <cstring>edtQualityLevel</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="2" colspan="3">
        <widget class="QLabel" name="lblUetAvarageMedWorker">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>УЕТ для среднего медицинского персонала</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QLabel" name="lblUetDoctor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>УЕТ для врача</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="11" column="3">
        <widget class="QDoubleSpinBox" name="edtUetChildAvarageMedWorker">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="4">
        <widget class="QComboBox" name="cmbLicense">
         <item>
          <property name="text">
           <string>не требуется</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>требуется лицензия</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>требуется персональный сертификат</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLabel" name="lblUetAdultAvarageMedWorker">
         <property name="text">
          <string>Взрослый</string>
         </property>
         <property name="buddy">
          <cstring>edtUetAdultAvarageMedWorker</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblBegDate">
         <property name="text">
          <string>Начальная дата</string>
         </property>
         <property name="buddy">
          <cstring>edtBegDate</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="edtBegDate">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>131</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblEndDate">
         <property name="text">
          <string>Конечная дата</string>
         </property>
         <property name="buddy">
          <cstring>edtEndDate</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="edtEndDate">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>131</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1" colspan="4">
        <widget class="QCheckBox" name="chkeisLegacy">
         <property name="text">
          <string>&amp;Унаследовано из ЕИС</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblLicense">
         <property name="text">
          <string>&amp;Лицензирование</string>
         </property>
         <property name="buddy">
          <cstring>cmbLicense</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblInfis">
         <property name="text">
          <string>Код &amp;ИНФИС</string>
         </property>
         <property name="buddy">
          <cstring>edtInfis</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="4">
        <widget class="QLineEdit" name="edtInfis"/>
       </item>
       <item row="13" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>48</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lblUetAdultDoctor">
         <property name="text">
          <string>Взрослый</string>
         </property>
         <property name="buddy">
          <cstring>edtUetAdultDoctor</cstring>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="lblUetChildDoctor">
         <property name="text">
          <string>Детский</string>
         </property>
         <property name="buddy">
          <cstring>edtUetChildDoctor</cstring>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QLabel" name="lblUetChildAvarageMedWorker">
         <property name="text">
          <string>Детский</string>
         </property>
         <property name="buddy">
          <cstring>edtUetChildAvarageMedWorker</cstring>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QDoubleSpinBox" name="edtUetChildDoctor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QDoubleSpinBox" name="edtUetAdultAvarageMedWorker">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lblSuperviseComplexityFactor">
         <property name="text">
          <string>Коэффициент сложности курации</string>
         </property>
         <property name="buddy">
          <cstring>edtSuperviseComplexityFactor</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="4">
        <widget class="QCheckBox" name="chkNomenclatureLegacy">
         <property name="text">
          <string>&amp;Унаследовано из номенклатуры Минздрава</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="lblServiceCategory">
         <property name="text">
          <string>Категория</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1" colspan="4">
        <widget class="CRBComboBox" name="cmbServiceCategory"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabContent">
      <attribute name="title">
       <string>Состав</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="CInDocTableView" name="tblServices"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMedicalAidProfiles">
      <attribute name="title">
       <string>Профили, виды и типы помощи</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblProfiles">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Профиль по умолчанию</string>
         </property>
         <property name="buddy">
          <cstring>cmbMedicalAidProfile</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="CRBComboBox" name="cmbMedicalAidProfile"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="CInDocTableView" name="tblMedicalAidProfiles"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblMedicalAidProfile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Тип по умолчанию</string>
         </property>
         <property name="buddy">
          <cstring>cmbMedicalAidType</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="CRBComboBox" name="cmbMedicalAidType"/>
       </item>
       <item row="1" column="1">
        <widget class="CRBComboBox" name="cmbMedicalAidKind"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblMedicalAidKind">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Вид по умолчанию</string>
         </property>
         <property name="buddy">
          <cstring>cmbMedicalAidKind</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPurpose">
      <attribute name="title">
       <string>Типы и цели случая лечения</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Цели случая лечения</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CInDocTableView" name="tblGoals"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="CRBComboBox" name="cmbCaseCast">
           <property name="minimumSize">
            <size>
             <width>448</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Типы случая лечения</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMKB">
      <attribute name="title">
       <string>МКБ</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="CInDocTableView" name="tblMKB"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbServiceGroup</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>chkeisLegacy</tabstop>
  <tabstop>cmbLicense</tabstop>
  <tabstop>edtInfis</tabstop>
  <tabstop>edtQualityLevel</tabstop>
  <tabstop>edtSuperviseComplexityFactor</tabstop>
  <tabstop>edtUetAdultDoctor</tabstop>
  <tabstop>edtUetChildDoctor</tabstop>
  <tabstop>edtUetAdultAvarageMedWorker</tabstop>
  <tabstop>edtUetChildAvarageMedWorker</tabstop>
  <tabstop>tblServices</tabstop>
  <tabstop>cmbMedicalAidProfile</tabstop>
  <tabstop>cmbMedicalAidKind</tabstop>
  <tabstop>cmbMedicalAidType</tabstop>
  <tabstop>tblMedicalAidProfiles</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
